.github/workflows/test.yml

```
@@ -1,73 +1,15 @@
 name: test workflow
 on:
   push:
-    branches:
-      - "*"
-      - "**/*"
+    branches: 
+      - main
   pull_request:
+    paths:
+      - crates/voicevox_core**
```

xtaskを含めてRustのクレートは全部チェックする必要があると思います。

```suggestion
      - crates/**
```